### PR TITLE
create saml doc

### PR DESCRIPTION
**Pull request by nataliejeremy**

### CODE REVIEW

File: source/docs/articles/organizations/saml-for-orgs.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: SAML for Orgs
+description: Learn about single-sign on.
+category:
+  - developing
+---
+
+Single-sign on allows your users to authenticate against your IdP when using the Pantheon product.
```
Comment (ari-gold) [inline source/docs/articles/organizations/saml-for-orgs.md]: "Single-sign on" should be "Single sign-on (SSO)"

source: http://en.wikipedia.org/wiki/Single_sign-on

File: source/docs/articles/organizations/saml-for-orgs.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+title: SAML for Orgs
```
Comment (ari-gold) [inline source/docs/articles/organizations/saml-for-orgs.md]: How about something a little more descriptive? Maybe "Single sign-on for Pantheon Organizations"

**Comment by nataliejeremy** [comment]
@ari-gold Thanks, I'll incorporate those edits. I'm still waiting for some other info so I'll update all at once.

**Comment by nataliejeremy** [comment]
@bmackinney please merge if this is good to go.

**Pull request merged**